scripts/st2bootstrap-deb.sh

```diff
@@ -317,7 +317,28 @@ install_st2() {
     sudo apt-get install -y st2${ST2_PKG_VERSION}
   else
     sudo apt-get install -y jq
-    PACKAGE_URL="$(curl -Ss -q https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts | jq -r '.[].url' | egrep "${SUBTYPE}/st2_.*.deb")"
+
+    echo "Retrieving packages metadata from https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts"
```

Comment:
We'll need the same logic for `install_st2mistral()` function and `st2bootstrap-el6.sh` with `st2bootstrap-el7.sh` scripts as well.

Reply:
I can do that, but my eventual goal was to move all the common functionality in common.sh or similar and build final script from that (concatenating things together) as opposed to more copy and paste...

Reply:
For now, I will refactor this in a function, copy and paste it in all the files and work on the concatenation stuff (long overdue) once this PR is merged to make changes easier to review.